src/web/app/components/sortable-table/sortable-table.component.ts

```diff
@@ -53,25 +60,28 @@ export class SortableTableComponent implements OnInit, OnChanges {
   @Input()
   initialSortBy: SortBy = SortBy.NONE;
 
+  @Output()
+  sortEvent: EventEmitter<any> = new EventEmitter();
+
   columnToSortBy: string = '';
   sortOrder: SortOrder = SortOrder.ASC;
   tableRows: SortableTableCellData[][] = [];
 
-  constructor(private tableComparatorService: TableComparatorService) { }
+  constructor(private tableComparatorService: TableComparatorService) {}
 
   ngOnInit(): void {
-    this.tableRows = this.rows.slice(); // Shallow clone to avoid reordering original array
+    this.tableRows = this.rows;
```

Comment:
Not sure if this is a reasonable thing to do, but i need to mutate the original array to ensure that the row state are synced across `sortable-table`, `sessions-table` and `instructor-home-page`/`instructor-sessions-page`.

Reply:
Thanks - checked and didn't find any issues as well